00-Raster-Structure.Rmd

```diff
@@ -241,89 +223,94 @@ crs(DSM)
-sign. 
+ `+proj=utm +zone=18 +datum=WGS84 +units=m +no_defs +ellps=WGS84 +towgs84=0,0,0`
 
 We'll focus on the first few components of the CRS in this lesson.
 
-* `+proj=utm` The projection of this dataset is UTM 
-* `+zone=18` If you read more about UTM, you will learn that the UTM projection 
-divides up the world into zones. this data happens to be in zone 18 (Massachusettes).
-* `+datum=WGS84` The datum tells us how the center point was defined to flatten the data
-more on that HERE <<ADD LINK>>
-* `+units=m` This is the horizontal units that the data are in. Here we see the units 
-are meters. This means that if the data are 1x1 resolution, that each pixel represents
-a 1 x 1 meter area on the ground.
+* `+proj=utm` The projection of the dataset. Our data are in UTMs.  
+* `+zone=18` The UTM projection divides up the world into zones, this element
+tells you which zone the data is in.  Harvard Forest is within Zone 18.
+* `+datum=WGS84` Which datum was used to define the center point of the 
+projection. Our data are within WGS84 the most common projection currently in 
+use in the US. 
+* `+units=m` This is the horizontal units that the data are in. Our units 
+are meters. 
 
-![UTM Coordinate system](http://upload.wikimedia.org/wikipedia/en/thumb/5/57/Utm-zones.svg/720px-Utm-zones.svg.png)
+## Resolution
+A raster has horizontal (x and y) resolution. This resolution represents the 
+area on the ground that each pixel covers. We will need to know the units used 
+to calculate the resolution. In our data the units are meters. This means that 
+if the data are 1x1 resolution, that each pixel represents a 1 x 1 meter area
+on the ground.
 
+>Bonus: We can find out the units using either of two methods: viewing the 
+>coordinate reference system string `crs()` like we just did or by `@data`. 
+>If you enter `DSM@data`, you'll see that units are not specified for these data.
 
 #Calculate the Min and Max values for the raster
+Dependig on our question we may want to know the min or max value of the data 
+in our raster.  Let's say we are thinking about studying a species of tree 
+that grows only above 600m. Therefore, we want to know if this might be a good
+species to monitor the phenology of within this area of the Harvard Forest. 
+We are unsure of the elevation range within the area. 
 
-Sometimes the raster statistics including the min and max values have already been
-calculated for us. Other times they have not. If they are not calculated you can
-for R to calculate them using the `setMinMax` function. Once they are calculated 
-you can then call them using the same `@data` syntax that we used above to look
-at the units.
+Raster statistics can include the min and max values, however, they aren't
+always caluclated and included. If they are not calculated, you can calculate
+them using the `setMinMax()` function.
 
 ```{r set-min-max }
+#Our raster already has these values calculated. Slot "min" and Slot "max":
```

Comment:
Issue https://github.com/data-lessons/NEON-R-Spatial-Raster/issues/26
 I'll leave for now and we can address later, especially if it fits better elsewhere.